File: mario.py
from cs50 import get_int

while True:
    height = get_int("Height: \n");
    if height >= 1 and height <= 8:
        break

for i in range(1, height + 1):
    print(" " * (height - i), end = "")
    print("#" * i, end = "")
    print("  ", end = "")
    print("#" * i)

File: mario.c
#include <stdio.h>
#include <cs50.h>

int main(void) 
{
    do 
    {
     int height = get_int("Height: \n");    
    } while(height >= 1 && height <= 8) 
    for (int i = 1; i <= height; i++)
    {
        for(int j = 0; j < (height - i); j++) 
        {
        printf(" ");    
        }
        for(int k = 0; k < i; k++) 
        {
        printf("#");    
        }
        printf("  ");    
        for(int k = 0; k < i; k++) 
        {
        printf("#");    
        }
        printf("\n");    
    }
}